dags/glam_glean_imports.py

```diff
@@ -131,16 +131,16 @@
             [wait_for_fenix, wait_for_fog] >> glean_env_task_group
 
 
-default_glam_import_image = "gcr.io/moz-fx-dataops-images-global/gcp-pipelines/glam/glam-production/glam:2023.07.1-43"
+default_glam_import_image = "gcr.io/moz-fx-dataops-images-global/gcp-pipelines/glam/glam-production/glam:2024.10.0-58"
 
 base_docker_args = ["/venv/bin/python", "manage.py"]
 
 for env in ["Dev", "Prod"]:
     glam_import_image = default_glam_import_image
     if env == "Dev":  # noqa 114
-        glam_import_image = "gcr.io/moz-fx-dataops-images-global/gcp-pipelines/glam/glam-production/glam:2023.07.1-43"
+        glam_import_image = "gcr.io/moz-fx-dataops-images-global/gcp-pipelines/glam/glam-production/glam:2024.10.0-58"
     elif env == "Prod":
-        glam_import_image = "gcr.io/moz-fx-dataops-images-global/gcp-pipelines/glam/glam-production/glam:2023.07.1-43"
+        glam_import_image = "gcr.io/moz-fx-dataops-images-global/gcp-pipelines/glam/glam-production/glam:2024.10.0-58"
```

Comment:
Why is this needed when all the images are the same?

Reply:
GLAM has different psql dbs for dev and prod, so this would allow for testing of more complicated things. That said, I don't think I've ever seen it being used.